phoenix-core/src/it/java/org/apache/phoenix/end2end/InListIT.java

```diff
@@ -73,7 +74,8 @@ public void setup() throws Exception {
     }
 
     @After
-    public void cleanUp() throws SQLException {
+    public void cleanUp() throws Exception {
+        confirmStoreRefCountLeak();
         deleteTenantData(descViewName);
```

Comment:
The should probably go into a finally as well.

Reply:
This too should be taken care of by BaseTest#confirmStoreRefCountLeak() because it internally frees up resources and while doing so, drops all tables before shutting down cluster.